eos-migrate-chromium-profile

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python3
+#
+# eos-migrate-chromium-profile: move users' Chromium profile to its new home
+#
+# This script is based on eos-migrate-firefox-profile.
```

Comment:
A bit of a sad amount of copypasta but I guess we can live with it for the upside of these being simple scripts.

Reply:
Yeah, exactly.